src/components/Homepage_image/Homepage_image.js

```diff
@@ -0,0 +1,28 @@
+import { getCurrentGameMode } from "../gameMenu";
+import { PHOTO_MODE, NAME_MODE, FAMILY_NAME_MODE } from "../API/newQuestion";
+
+export const homePagePoster = () => {
+  const poster = document.createElement("img");
+  poster.setAttribute("src", "../photos/game-of-thrones-poster.png");
```

Comment:
It can't be another photo. Use one of the photos from the modes or use this photo as default.